src/v/cluster/config_manager.cc

```diff
@@ -657,11 +657,6 @@ void config_manager::merge_apply_result(
   config_status& status,
   cluster_config_delta_cmd_data const& data,
   apply_result const& r) {
-    vlog(
-      clusterlog.trace,
-      "merge_apply_result: data {} {}",
-      fmt::ptr(&data),
```

Comment:
Good call removing this. Nothing secret revealed here, but I'm not sure how useful this actually is (maybe if you're in a debugger or something).

Reply:
Yeah, this was something left behind from ancient debugging that I was cleaning up while making the pass.